components/bl-sweet-alert/README.md

```diff
@@ -0,0 +1,156 @@
+| Allow Escape Key <br> `allowEscapeKey`        | *Checkbox*                                                                                                                    | `true`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |                           | NO           | YES        | Controls whether the user can dismiss the popup by pressing the `Esc` key by default or not.                                                                                                              |
+| Allow Enter Key <br> `allowEnterKey`          | *Checkbox*                                                                                                                    | `true`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |                           | NO           | YES        | Controls whether the user can confirm the popup by pressing the `Enter` or `Space` keys by default or not, unless they manually focus the confirm button.                                                 |
+| Buttons Settings <br>                         | *JSON*                                                                                                                        | `{ "showConfirmButton": true, "confirmButtonText": "OK", "confirmButtonColor": "#3085d6", "confirmButtonAriaLabel": "", "focusConfirm": true, "showDenyButton": false, "denyButtonText": "No", "denyButtonColor": "#dd6b55", "denyButtonAriaLabel": "", "focusDeny": false, "showCancelButton": false, "cancelButtonText": "Cancel", "cancelButtonColor": "#aaa", "cancelButtonAriaLabel": "", "focusCancel": false, "showCloseButton": false, "closeButtonHtml": "&times;", "closeButtonAriaLabel": "Close this dialog" }` | Buttons Settings Logic    | YES          | YES        | This logic allows to specify the settings on the "Confirm", "Deny", "Cancel", "Close" buttons.                                                                                                            |
```

Comment:
JSON in table looks unreadable. Change it to link and past as code below the table.

![image](https://github.com/Backendless/ui-builder-library/assets/15347646/56826925-ea6a-4bd7-b75d-30500a2876b8)